integrationtests/python_src/px4_it/mavros/mission_test.py

```diff
@@ -303,6 +348,19 @@ def test_mission(self):
 
             index += 1
 
+        rospy.loginfo("mission done, calculating performance metrics")
+        last_log = get_last_log()
+        rospy.loginfo("log file %s", last_log)
+        data = px4tools.ulog.read_ulog(last_log).resample_and_concat(0.1)
+        res = px4tools.estimator_analysis(data, False)
+
```

Comment:
This is the key section where I set the requirements for the test to pass in terms of error from ground truth.